Source/Fuse.Reactive.Bindings/Instance.API.uno

```diff
@@ -0,0 +1,356 @@
+using Uno;
+using Uno.Collections;
+using Uno.UX;
+
+using Fuse.Internal;
+
+namespace Fuse.Reactive
+{
+	public enum InstanceDefer
+	{
+		/** Items will be added immediately to the tree. */
+		Immediate,
+		/** Item changes will be buffered and added once per frame. */
+		Frame,
+		/** Items will be added as though they are wrapped in a @Deferred node. */
+		Deferred,
+	}
+	
+	public enum InstanceReuse
+	{
+		/** Instances are not reused */
+		None,
+		/** Instances can be reused in the same frame */
+		Frame,
+	}
+	
+	public enum InstanceObjectMatch
```

Comment:
Please add docs for the enum itself too. If it is only used one place, please point to the place it is used for docs.